src/styles/types.ts

```diff
@@ -612,17 +639,19 @@
           "default": string,
           "active": string,
           "disabled-checked": string,
```

Comment:
Can we rename it as disabled instead of disabled-checked or unchecked

Reply:
They're different states though, are we able to indicate the difference between `disabled + checked` and `disabled + unchecked` without them?

Reply:
yes we are able to differentiate with the data attribute  `[data-state] | "checked" \| "unchecked" \| "indeterminate"`

Reply:
I asked because we have removed the disabled-unchecked from tokens